src/EditorFeatures/Core/Implementation/LineSeparators/AdornmentManager.cs

```diff
@@ -302,6 +316,29 @@ private void UpdateInvalidSpans()
             }
         }
 
+        private NormalizedSpanCollection GetCollapsedSpans()
+        {
+            var elisionSnapshot = _textView.VisualSnapshot as IProjectionSnapshot;
```

Comment:
Even in cshtml, workflow, etc?  Is there a way we can at least test that?

If it is one of those other things, are source spans actually in buffers that we know/care about?